qemu/tests/chardev_tls_encryption.py

```diff
@@ -33,6 +34,9 @@ def run(test, params, env):
     """
     clean_cmd = params["clean_cmd"]
     try:
+        pkgs = params.objects("depends_pkgs").split()
```

Comment:
`params.objects` will return a list.
```python
    def objects(self, key):
        """
        Return the names of objects defined using a given key.

        :param key: The name of the key whose value lists the objects
                (e.g. 'nics').
        """
        lst = self.get(key, "").split()
        # remove duplicate elements
        objs = list({}.fromkeys(lst).keys())
        # sort list to keep origin order
        objs.sort(key=lst.index)
        del lst
        return objs
```

Reply:
Thanks for reminder @PaulYuuu You and yanan all give the suggestions, do you mind to tell which one is better ?

Reply:
Yeah, `objects` is more simple~

Reply:
@MiriamDeng  No need to use `split` if use `objects`, it is a list already.

Reply:
So I just need to use "objects" here, and code is like, pkgs = params.objects("depends_pkgs"), thanks